projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-color.service.spec.ts

```diff
@@ -27,13 +27,10 @@ describe('DataLayerColorService', () => {
       expect(setColorSpy).toHaveBeenCalled();
     }));
   });
-  describe('getAnnotationColor', () => {
+  describe('addTransparencyAbritraryColor', () => {
     it('should return the annotation background color', inject([DataLayerColorService], (service: DataLayerColorService) => {
```

Comment:
please update this test description to match the new behavior

Reply:
Yes, I am updating this test description to match the new behavior.